version

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Version script for Marathon.
+#
+# Change MINOR to x+1 and BRANCH_POINT to commit hash of common ancestor of master and releases/1.x
+# after a releases/1.x was cut
+MAJOR=1
+MINOR=6
+BRANCH_POINT=aa89e89
+
+# Infer version
+# Number of commits since branch point
+BUILD_NUMBER="$(git rev-list --count --first-parent $BRANCH_POINT..HEAD)"
+
+# E.g. 1.7.42
+echo "$MAJOR.$MINOR.$BUILD_NUMBER"
```

Comment:
Why is a patch number a build number? Is it as intended that each commit increments the patch number?

Reply:
I'll change it to commit number. And yes, it increments with each commit.